sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md

```diff
@@ -89,6 +96,24 @@ rename-mapping:
   Scope: BenefitRecommendationScope
   Grain: BenefitRecommendationUsageGrain
   OperatorType: ComparisonOperatorType
+  DismissAlertPayload.properties.closeTime: -|datetime
+  DismissAlertPayload.properties.statusModificationTime: -|datetime
```

Comment:
consider renaming to ModifiedOn

Reply:
renaming to ModifiedOn will throw 'Sequence contains more than one matching element '
now rename it to `StatusModifiedOn`